client/src/containers/CampaignAdmin/CampaignAdminList.js

```diff
@@ -444,126 +379,145 @@ class CampaignAdminList extends Component {
+        itemLinks = item._links;
+      }
 
-      const wrapperClassnames = classNames('list-group-wrapper', {
-        'list-group-wrapper--empty': listGroupItems.length === 0,
+      // Add extra css class for published items
+      const itemClassNames = classNames({
+        'list-group-item--inactive': (item.ChangeType === 'none' || campaign.State === 'published'),
+        active: (selected),
       });
 
-      // Merge into group
-      accordionBlocks.push(
-        <div className={wrapperClassnames} key={groupid}>
-          <AccordionBlock groupid={groupid} title={title}>
-            {
-              listGroupItems.length > 0
-                ? listGroupItems
-                : <p className="list-group-item">{group.noItemsText}</p>
-            }
-          </AccordionBlock>
-        </div>
+      let isLinked = !!selectedItemsLinkedTo.find(
+        linkToObj => linkToObj.ChangeSetItemID === parseInt(item.ID, 10));
+
+      isLinked = isLinked || selectedItemsLinkedFrom.find(linkFromObj => (
+        linkFromObj.ChangeSetItemID === item.ID
+      ));
+
+      listGroupItems.push(
+        <ListGroupItem
+          key={item.ID || index}
+          className={itemClassNames}
+          onClick={handleItemSelected}
+          onClickArg={item.ID}
+        >
+          <CampaignAdminItem
+            item={item}
+            campaign={record}
+            selected={selected}
+            isLinked={isLinked}
+          />
+        </ListGroupItem>
       );
     });
 
-    const body = <Accordion>{accordionBlocks}</Accordion>;
-
-    const loading = this.props.loading && [
-      <div key="overlay" className="cms-content-loading-overlay ui-widget-overlay-light" />,
-      <div key="spinner" className="cms-content-loading-spinner" />,
-    ];
+    const wrapperClassnames = classNames('list-group-wrapper', {
+      'list-group-wrapper--empty': listGroupItems.length === 0,
+    });
 
-    return (
-      <div className={`fill-width campaign-admin__campaign ${selectedClass}`}>
-        {loading}
-        {this.renderCampaignAdminListDetail(body, itemLinks)}
-        {this.renderPreview(itemLinks, itemId)}
+    // Merge into group
+    accordionBlocks.push(
+      <div className={wrapperClassnames} key={groupid}>
+        <AccordionBlock groupid={groupid} title={title}>
+          {
+            listGroupItems.length > 0
+              ? listGroupItems
+              : <p className="list-group-item">{group.noItemsText}</p>
+          }
+        </AccordionBlock>
       </div>
     );
-  }
-}
+  });
+
+  const body = <Accordion>{accordionBlocks}</Accordion>;
+
+  const loadingSpinner = !loading && [
+    <div key="overlay" className="cms-content-loading-overlay ui-widget-overlay-light" />,
+    <div key="spinner" className="cms-content-loading-spinner" />,
+  ];
+
+  return (
+    <div className={`fill-width campaign-admin__campaign ${selectedClass}`}>
+      {loadingSpinner}
+      {renderCampaignAdminListDetail(body)}
```

Comment:
`itemLinks` argument was removed! Looks like there's no param to take it in the function being called so it's a correct change - please just remember to point out why changes like this are made.